README.md

```diff
@@ -2,8 +2,16 @@
 
 [![Build Status](https://travis-ci.org/puppetlabs/trapperkeeper-status.svg)](https://travis-ci.org/puppetlabs/trapperkeeper-status)
 
-A trapperkeeper service for getting the status of other trapperkeeper
-services.
+A Trapperkeeper service that provides a web endpoint for getting status
+information about a running Trapperkeeper application.
```

Comment:
I think this should pluralized, s/a running Trapperkeeper application/running Trapperkeeper applications